tests/common.go

```diff
@@ -124,6 +129,9 @@ func InitInstance() {
 
 	err = Inst().V.Init(Inst().S.String(), Inst().N.String(), token, Inst().Provisioner)
 	expect(err).NotTo(haveOccurred())
+
+	err = Inst().B.Init(Inst().S.String(), Inst().N.String(), Inst().V.String(), token)
```

Comment:
The backup server won't be running in all the tests, so this should be done only if a driver was provided.

Reply:
Fixed.